data/events/2022-london.yml

```
@@ -56,6 +56,7 @@ team_members: # Name is the only required field for team members.
     twitter: "rjw1"
     website: "https://randomness.org.uk"
   - name: "Chris Mills"
+    github: "chriscoffee"
   - name: "Dan Webb"
     twitter: "dan_webb"
```

```suggestion
    twitter: "dan_webb"
    github: damacus
```


If we're adding these in you can have my GitHub too